lib/src/key/cf.rs

```diff
@@ -0,0 +1,132 @@
+use derive::Key;
+use serde::{Deserialize, Serialize};
+
+use crate::vs;
+
+use std::str;
+
+// Cf stands for change feeds
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Key)]
+pub struct Cf {
+	__: u8,
+	_a: u8,
+	pub ns: String,
+	_b: u8,
+	pub db: String,
+	_d: u8,
+	_e: u8,
+	_f: u8,
+	// vs is the versionstamp of the change feed entry that is encoded in big-endian.
+	// Use the to_u64_be function to convert it to a u128.
+	pub vs: [u8; 10],
+	_c: u8,
+	pub tb: String,
```

Comment:
We may want to move to a zero-copy approach for the keys, avoiding copying ns, db, tb.

The other keys have been migrated to this model:

https://github.com/surrealdb/surrealdb/blob/73a4e54ad5f784420f9bc3ce2246e13920a55788/lib/src/key/tb.rs#L5

Reply:
Good to know! Thanks!
Addressed in e55f3e5e 👍